collect_app/src/main/java/org/odk/collect/android/widgets/ExStringWidget.java

```diff
@@ -116,7 +118,11 @@ protected void setUpLayout(Context context) {
         answerLayout.setOrientation(LinearLayout.VERTICAL);
         answerLayout.addView(launchIntentButton);
         answerLayout.addView(answerText);
-        addAnswerView(answerLayout);
+        
+        Resources resources = context.getResources();
+        int marginStandard = dpFromPx(context, resources.getDimensionPixelSize(R.dimen.margin_standard));
+        int margin = marginStandard - FIELD_HORIZONTAL_MARGIN_MODIFIER;
```

Comment:
This margin ends up being too big as `QuestionWidget#getSimpleButton` is adding a margin on for us. Your changes should allow us to delete the code that adds the margin there so you should be able to do that! Again though, I'd wait for #3483 to be merged as this method moves to `WidgetViewUtils`.